ansible/roles/test/files/ptftests/utilities.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utility functions can re-used in testing scripts.
+"""
+import re
+
+
+def _parse_column_positions(self, sep_line, sep_char='-'):
+    prev = ' ',
+    positions = []
+    for pos, char in enumerate(sep_line + ' '):
+        if char == sep_char:
+            if char != prev:
+                left = pos
+        else:
+            if char != prev:
+                right = pos
+                positions.append((left, right))
+        prev = char
+    return positions
+
+
+# sync from sonic.py SonicHost.parse_show
+def parse_show(self, output_lines):
+    result = []
+
+    sep_line_pattern = re.compile(r"^( *-+ *)+$")
+    sep_line_found = False
+    for idx, line in enumerate(output_lines):
+        if sep_line_pattern.match(line):
+            sep_line_found = True
+            header_line = output_lines[idx-1]
+            sep_line = output_lines[idx]
+            content_lines = output_lines[idx+1:]
+            break
+
+    if not sep_line_found:
+        self.log('Failed to find separation line in the show command output')
+        return result
+
+    try:
+        positions = self._parse_column_positions(sep_line)
```

Comment:
Same for all `self` usage.

Reply:
In this function self is not needed, I will remove it in next diff. Thanks for point it out.